src/deltacode/utils.py

```diff
@@ -25,13 +25,15 @@
 
 from __future__ import absolute_import, division
 
+from bitarray import bitarray
 from collections import defaultdict
+from bitarray import bitdiff
 
+import binascii
 import os
 
-
 from commoncode import paths
-
+from scancode import cli_test_utils
```

Comment:
Why are we importing this in the source file? Also why are we importing from scancode when you are copying over functions to deltacode?